Podfile

```diff
@@ -17,7 +17,7 @@ end
 
 target 'GliaWidgets' do
   pod 'PureLayout', '~> 3.1'
-  pod 'SalemoveSDK'
+  pod 'SalemoveSDK', :path => '../ios-sdk/GliaCoreSDK.sources.podspec'
```

Comment:
This was changed due to testing purposes, yes? We should be able to revert it now.

Reply:
yes